run

```diff
@@ -177,8 +204,8 @@ case $run_command in
 
     # Run watch command in the background
     if [[ ${1:-} == "--watch" ]] || [[ ${1:-} == "-w" ]]; then
-      container_name="yarn-$(date +'%s')"
-      trap 'docker kill ${container_name}' EXIT
+      container_name="${project}-yarn-$(date +'%s')"
```

Comment:
For consistency with the others, this should probably be `yarn-watch`, rather than `yarn $(date ...)`